doc/sphinx/language/extensions/arguments-command.rst

```diff
@@ -0,0 +1,442 @@
+       clears argument scopes of :n:`@smart_qualid`
+     `extra scopes`
+       defines extra argument scopes, to be used in case of coercion to ``Funclass``
+       (see the :ref:`implicitcoercions` chapter) or with a computed type.
+     `simpl nomatch`
+       prevents performing a simplification step for :n:`@smart_qualid`
+       that would expose a match construct in the head position.  See :ref:`Args_effect_on_unfolding`.
+     `simpl never`
+       prevents performing a simplification step for :n:`@smart_qualid`.  See :ref:`Args_effect_on_unfolding`.
+
+     `clear bidirectionality hint`
+       removes the bidirectionality hint, the `&`
+
+     :n:`@implicits_alt`
+       use to specify alternative implicit argument declarations
+       for functions that can only be
+       applied to a fixed number of arguments (excluding, for instance,
+       functions whose type is polymorphic).
+       For parsing, the longest list of implicit arguments matching the function application
+       is used to select which implicit arguments are inserted.
+       For printing, the alternative with the most implicit arguments is used; the
+       implict arguments will be omitted if :flag:`Printing Implicit` is not set.
+       See the example :ref:`here<example_more_implicits>`.
+
+       .. todo the above feature seems a bit unnatural and doesn't play well with partial
+          application.  See https://github.com/coq/coq/pull/11718#discussion_r408841762
+
+   Use :cmd:`About` to view the current implicit arguments setting for a :token:`smart_qualid`.
+
+   Or use the :cmd:`Print Implicit` command to see the implicit arguments
+   of an object (see :ref:`displaying-implicit-args`).
+
+Manual declaration of implicit arguments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. example::
+
+    .. coqtop:: reset all
+
+       Inductive list (A : Type) : Type :=
+       | nil : list A
+       | cons : A -> list A -> list A.
+
+       Check (cons nat 3 (nil nat)).
+
+       Arguments cons [A] _ _.
+
+       Arguments nil {A}.
+
+       Check (cons 3 nil).
+
+       Fixpoint map (A B : Type) (f : A -> B) (l : list A) : list B :=
+         match l with nil => nil | cons a t => cons (f a) (map A B f t) end.
+
+       Fixpoint length (A : Type) (l : list A) : nat :=
+         match l with nil => 0 | cons _ m => S (length A m) end.
+
+       Arguments map [A B] f l.
+
+       Arguments length {A} l. (* A has to be maximally inserted *)
+
+       Check (fun l:list (list nat) => map length l).
+
+.. _example_more_implicits:
+
+.. example:: Multiple alternatives with :n:`@implicits_alt`
+
+   .. coqtop:: all
+
+       Arguments map [A B] f l, [A] B f l, A B f l.
+
+       Check (fun l => map length l = map (list nat) nat length l).
+
+.. _auto_decl_implicit_args:
+
+Automatic declaration of implicit arguments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   The :n:`default implicits @args_modifier` clause tells |Coq| to automatically determine the
```

Comment:
Should the quotation of `args_modifier` be separated from the quotation of `default implicits`?  Otherwise it reads to me as if it's part of the syntax of `default implicits` rather than describing the kind of thing it is.

Reply:
I add double quotes around "default implicits"